CODEOWNERS

```diff
@@ -0,0 +1,32 @@
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+
+# SECTION: Owner(s) for everything in the repo, unless a later match takes precedence
+# FILE PATTERN                              OWNER(S)
+*                                           @saltstack-formulas/wg
```

Comment:
@dafyddj In terms of the conversion script, nothing fancy is needed.  All we need to do is change this line to:

```diff
-*                                           @saltstack-formulas/wg
+*                                           @NONE
```

Reply:
Ok, I can take a look at that.